tests/unit/models/reddit/test_emoji.py

```diff
@@ -1,5 +1,6 @@
 import pickle
 
+import pytest
```

Comment:
There should probably be another newline after this import.

Reply:
The test imports don't really matter imo

Reply:
From the perspective of this tests package, praw.models is technically a third-party package. I think that's pretty consistent throughout the tests.

Reply:
In that case, for alphabetization's sake, `pytest` should be last, no? Either way I'm splitting hairs though.

Reply:
import X type imports go before `from X import Y` type imports. So I think everything is in order here. Unless I'm looking at the wrong view.